libr/include/r_bin.h

```diff
@@ -303,6 +303,7 @@ typedef struct r_bin_section_t {
 	ut64 vaddr;
 	ut64 paddr;
 	ut32 perm;
+	ut32 flags;
```

Comment:
This change breaks the ABI so we can't merge this until the 5.9.8 release is out. we can have R2_USE_NEW_ABI ifdef if you want.. but personall i would wait because it shuoldnt be taking long and we can probably introduce this new field to work with other fileformats

Reply:
for ELF https://www.sco.com/developers/gabi/2003-12-17/ch4.sheader.html